.github/workflows/build.yaml

```diff
@@ -16,12 +16,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2019, macos-latest] # Stay on 2019 for Windows SDK Version 10.0.16299.0
-
+        os: [windows-latest, macos-latest]
     steps:
       - uses: actions/checkout@v3
       - uses: microsoft/setup-msbuild@v1.1
         if: startsWith(matrix.os, 'windows')
+      - name: Install Windows SDK Version 10.0.16299.0
+        uses: GuillaumeFalourd/setup-windows10-sdk-action@v1.11
```

Comment:
Can we stay on windows-2019 and avoid running this instead?

If not, once you review what this action does, can we ping the commit sha here instead of `@v1.11` ?

Reply:
windows-2019 doesn't support the latest Android targets.
I checked the action and it's just a single powershell script that does the following:
```
download https://download.microsoft.com/download/F/8/E/F8EC2F14-375D-4C2E-9C2D-51CA8F7F590B/16299.15.170928-1534.rs3_release_WindowsSDK.iso
mount iso
run WinSDKSetup.exe
Start-Process with the following args "/features OptionId.UWPCpp /q"
where $WindowsSDKOptions is  @("OptionId.UWPCpp")
```
Since it's a single file, it may make more sense to just leave the script inside of the project? This way other contributors could also make use of it if needed.

Reply:
Yeah feel free to copy the steps in